client/src/styles/pages/_Profile.scss

```diff
@@ -6,8 +6,11 @@ main#profile {
     text-align: center;
   }
 
-  & > .avatar {
-    margin: auto;
+  .profile-avatar {
+    position: relative;
+    &-image > .avatar {
```

Comment:
Is `&-image` translated to `.profile-avatar-image`? That's awesome!

Reply:
Yeah !